adapter/src/dbt/adapters/fal_experimental/adapter.py

```diff
@@ -33,29 +40,43 @@ def run_with_adapter(code: str, adapter: BaseAdapter, config: RuntimeConfig) ->
             write_df=prepare_for_adapter(adapter, write_df_to_relation),
         )
 
-def _isolated_runner(code: str, config: RuntimeConfig, manifest: Manifest, macro_manifest: MacroManifest) -> Any:
+
+def _isolated_runner(
+    code: str,
+    config: RuntimeConfig,
+    manifest: Manifest,
+    macro_manifest: MacroManifest,
+    local_packages: Optional[bytes] = None,
+) -> Any:
     # This function can be run in an entirely separate
     # process or an environment, so we need to reconstruct
     # the DB adapter solely from the config.
     adapter = reconstruct_adapter(config, manifest, macro_manifest)
+    fal_scripts_path = get_fal_scripts_path(config)
+    if local_packages is not None:
+        assert not fal_scripts_path.exists()
```

Comment:
NIT: add an explanation
```suggestion
        assert not fal_scripts_path.exists(), "fal-scripts-path directory exists in environment already"
```